config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -131,6 +131,77 @@ periodics:
       secret:
         secretName: k8s-triage-robot-github-token
 
+- name: ci-k8s-triage-robot-close-staging-prs
+  interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: Closes staging repo PRs
+    testgrid-tab-name: close-staging-prs
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20240210-3c4f626498
+      command:
+      - commenter
+      args:
+      - |-
+        --query=is:pr
+        is:open
+        label:do-not-merge/blocked-paths
+        repo:kubernetes/api
+        repo:kubernetes/apiextensions-apiserver
+        repo:kubernetes/apimachinery
+        repo:kubernetes/apiserver
+        repo:kubernetes/cli-runtime
+        repo:kubernetes/client-go
+        repo:kubernetes/cloud-provider
+        repo:kubernetes/cluster-bootstrap
+        repo:kubernetes/code-generator
+        repo:kubernetes/component-base
+        repo:kubernetes/component-helpers
+        repo:kubernetes/controller-manager
+        repo:kubernetes/cri-api
+        repo:kubernetes/csi-translation-lib
+        repo:kubernetes/dynamic-resource-allocation
+        repo:kubernetes/endpointslice
+        repo:kubernetes/kms
+        repo:kubernetes/kube-aggregator
+        repo:kubernetes/kube-controller-manager
+        repo:kubernetes/kube-proxy
+        repo:kubernetes/kube-scheduler
+        repo:kubernetes/kubectl
+        repo:kubernetes/kubelet
+        repo:kubernetes/legacy-cloud-providers
+        repo:kubernetes/metrics
+        repo:kubernetes/mount-utils
+        repo:kubernetes/pod-security-admission
+        repo:kubernetes/sample-apiserver
+        repo:kubernetes/sample-cli-plugin
+        repo:kubernetes/sample-controller
+      - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
+      - |-
+        --comment=The Kubernetes project doesn't accept direct changes directly against this repository as outlined above.
```

Comment:
It feels kinda weird to duplicate this text in the blockade comment AND the robo-/close comment : thinking :

Reply:
"Closing per above"? (if you're confident the message will always be present)